.travis.yml

```diff
@@ -9,17 +9,25 @@ env:
   - SOURCE_FOLDER=tradingdb
   - PIP_USE_MIRRORS=true
   - DJANGO_READ_DOT_ENV_FILE=True
+  - DOCKER_COMPOSE_VERSION=1.24.0
```

Comment:
Travis' default docker-compose doesn't support templates v3.6, thus we have to upgrade it to a newer version.